code/game/machinery/portable_turret.dm

```diff
@@ -1044,6 +1044,10 @@ GLOBAL_LIST_EMPTY(turret_icons)
 /obj/machinery/porta_turret/syndicate/CanPass(atom/A)
     return ((stat & BROKEN) || !isliving(A))
 
+/obj/machinery/porta_turret/syndicate/CanPathfindPass(obj/item/card/id/ID, to_dir, atom/movable/caller, no_id = FALSE)
+	. = ..()
```

Comment:
Since you're not using the return value, the parent call isn't actually doing anything here.
```suggestion
```